src/tests/test_process.py

```diff
@@ -34,8 +53,22 @@ def setUpClass(cls):
         """Setup environment and start pybitmessage"""
         cls.home = os.environ['BITMESSAGE_HOME'] = tempfile.gettempdir()
         put_signal_file(cls.home, 'unittest.lock')
+        starttime = time.time()
         subprocess.call(cls._process_cmd)  # nosec
-        time.sleep(5)
+        timeout = starttime + 30.0
+        while time.time() <= timeout:
+            try:
+                pstat = os.stat(os.path.join(cls.home, 'singleton.lock'))
+                if starttime <= pstat.st_mtime and pstat.st_size > 0:
+                    break
+            except OSError:
+                pass
+            time.sleep(1)
+        if time.time() >= timeout:
+            raise OSError("Timeout starting {}".format(cls._process_cmd))
```

Comment:
https://travis-ci.org/github/g1itch/PyBitmessage/jobs/738338187

Reply:
That can be fixed later. We don't have a working OSX at the moment anyway.